From: Martin Gräßlin <mgraesslin@kde.org>
Date: Wed, 03 Jun 2015 15:37:28 +0000
Subject: Remove querying the indirect rendering context
X-Git-Url: http://quickgit.kde.org/?p=kinfocenter.git&a=commitdiff&h=9f9814219d899a64dd5b68711534ec7206858147
---
Remove querying the indirect rendering context

Mesa fails to make an indirect rendering context current and freezes
in the next X11 call.

Removing indirect rendering support seems like a workaround for the
problem, but realisticly it's no longer interesting what indirect
rendering provides if direct rendering is available. If direct
rendering is not available, it will pick indirect instead.

BUG: 344971
FIXED-IN: 5.4.0
REVIEW: 123995
---


--- a/Modules/opengl/opengl.cpp
+++ b/Modules/opengl/opengl.cpp
@@ -933,8 +933,6 @@
 
             l2 = get_gl_info_glx(dpy, scrnum, true, l1, l2);
             if (l2) l2->setExpanded(true);
-
-            if (IsDirect) l2 = get_gl_info_glx(dpy, scrnum, false, l1, l2);
 #endif
 #if KCM_HAVE_EGL
             l2 = get_gl_info_egl(dpy, scrnum, l1, l2);

